spec/rubocop/cop/rspec/predicate_spec.rb

```diff
@@ -0,0 +1,116 @@
+# encoding: utf-8
+
+describe RuboCop::Cop::RSpec::Predicate, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'when EnforcedStyle is boolean' do
+    let(:cop_config) { { 'EnforcedStyle' => 'equality' } }
+
+    TESTS = {
```

Comment:
Using a constant here ends up emitting warnings. If you use a local variable you'll get the same reuse perk without the warnings